client/src/models/api/stardust/ITransactionBlockResponse.ts

```diff
@@ -0,0 +1,9 @@
+import { IBlock } from "@iota/iota.js-stardust";
+import { IResponse } from "../IResponse";
+
+export interface ITransactionBlockResponse extends IResponse {
+    /**
+     * The transaction included block.
+     */
+    transactionBlock?: IBlock;
```

Comment:
Rename field to simply `block`. Transaction block is confusing.

Reply:
Resolved.